package.json

```diff
@@ -160,12 +161,15 @@
         "vscode": "^1.1.21"
     },
     "dependencies": {
-        "@octokit/rest": "^15.10.0",
+        "@octokit/rest": "15.10.0",
         "adm-zip": "^0.4.11",
         "chokidar": "^2.0.2",
         "fs-extra": "^7.0.0",
         "https-proxy-agent": "^2.1.1",
         "lockfile": "^1.0.4",
-        "temp": "^0.8.3"
+        "temp": "^0.8.3",
+        "ts-loader": "^5.2.1",
+        "webpack": "^4.20.2",
+        "webpack-cli": "^3.1.2"
```

Comment:
`ts-loader`, `webpack` and `webpack-cli` should be moved to `devDependencies`

Reply:
I always forget that :(